File: src/main/java/com/devspace/conexfy/configurations/ConOpenApiConfig.java
package com.devspace.conexfy.configurations;

import org.springframework.context.annotation.Configuration;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;

@Configuration
@OpenAPIDefinition(
    info = @Info(
        title = "Connection Service DevSpace API ",
        version = "v1",
        description = "API for managing HTTP connections (CRUD and execution)"
    )
)
public class ConOpenApiConfig {

}

File: src/main/kotlin/com/devspace/conexfy/configurations/ConOpenApiConfig.kt
package com.devspace.conexfy.configurations

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info
import org.springframework.context.annotation.Configuration

@Configuration(proxyBeanMethods = false)
@OpenAPIDefinition(
    info = Info(
        title = "Connection Service DevSpace API ",
        version = "v1",
        description = "API for managing HTTP connections (CRUD and execution)"
    )
)
class ConOpenApiConfig
